velox/common/memory/HashStringAllocator.cpp

```diff
@@ -316,6 +316,46 @@ void HashStringAllocator::free(Header* _header) {
   } while (header);
 }
 
+//  static
+int64_t HashStringAllocator::offset(
+    Header* FOLLY_NONNULL header,
+    Position position) {
+  int64_t size = 0;
+  for (;;) {
+    bool continued = header->isContinued();
+    auto length = header->size() - (continued ? sizeof(void*) : 0);
```

Comment:
Just to double-check whether I understand it correctly, is `continued ? sizeof(void*) : 0` used here because multi-part non-continuous allocations use the last 8 bytes for pointers to the next block?